test/jdk/valhalla/valuetypes/ObjectMethodsViaCondy.java

```diff
@@ -24,8 +24,9 @@
 /*
  * @test
  * @summary Test ObjectMethods::bootstrap call via condy
+ * @modules java.base/jdk.internal.value:+open
```

Comment:
Is `+open` needed?   PrimitiveClass methods are all public.  `+open` is needed only for deep reflection.

Same comments apply to other tests.